### PR TITLE
This program get commit

### PR DESCRIPTION
**Pull request by DARSHANIJAIN**
This is a pull request

### CODE REVIEW

File: _posts/0000-01-02-dishu18.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (dishu18) [inline _posts/0000-01-02-dishu18.md:5]: Life is beautiful

Comment (dishu18) [inline _posts/0000-01-02-dishu18.md:5]: Life is beautiful

**Comment by DARSHANIJAIN** [comment]
close it

**Comment by DARSHANIJAIN** [comment]
This get committed

**Comment by DARSHANIJAIN** [comment]
commit

**Pull request closed without merging**